src/models/meldinger/meldinger.ts

```diff
@@ -51,7 +51,8 @@ export enum Temagruppe {
     Pensjon = 'PENS',
     Arbeid = 'ARBD',
     AndreSosiale = 'ANSOS',
-    ØkonomiskSosial = 'OKSOS'
+    ØkonomiskSosial = 'OKSOS',
+    Null = ''
```

Comment:
vil temagruppe noen gang være en tom streng..?

Reply:
For dokumentvarsel og oppgavevarsel så er ikkje desse knytta til temagruppe, og vil vere blank fra REST-tjenesta. Tom er ikkje ukjent, då dei i dagens løsning ikkje viser ved tom.
Feilhåndtering for ei ukjent temagruppe bør vere på plass då me ikkje har 100% kontroll over verdikjeda.

Reply:
høres bra ut! Så lenge dette representeres ved en tom string så funker det jo bra 👍